test/src/util/pelz_json_parser_test_suite.c

```diff
@@ -51,18 +51,12 @@ void test_encrypt_parser(void)
   unsigned int json_key_id_len = 19;
   const char *enc_data = "QUJDREVGR0hJSktMTU5PUFFSU1RVVldYWVoxMjM0NTY=\n";
   unsigned int enc_data_len = 45;
-  const char *dec_data = "SwqqSZbNtN2SOfKGtE2jfklrcARSCZE9Tdl93pggkIsRkY3MrjevmQ==\n";
```

Comment:
This will still be need to test the decryption request.

Reply:
Take a look at 344c145. I went in the other direction, totally removing the separate tests for ```encrypt_parser``` and ```decrypt_parser``` since they're not separate functions anymore. I think most of the important test coverage is still given in the tests that check the request decoder.